onnxruntime/test/providers/cpu/tensor/dynamic_slice_op_test.cc

```diff
@@ -22,12 +22,14 @@ TEST(DynamicSliceTest, dynamic_slice_varied_types) {
   test2.AddOutput <int64_t> ("output", {2,2}, {5LL,6LL,8LL,9LL});
   test2.Run();
 
+#ifndef USE_CUDA
```

Comment:
why is it specific for CUDA? it should also work for CPU, right? These tests will be run on GPU as well when build it with CUDA

Reply:
Because CUDA does not handle string?

---
In reply to: [246168573](https://github.com/Microsoft/onnxruntime/pull/286#discussion_r246168573) [](ancestors = 246168573)

Reply:
It is excluding string from CUDA.